admin_manual/maintenance/migrating.rst

```diff
@@ -2,199 +2,235 @@
+
+- Ubuntu 16.04
+- SSH with ``PermitRootLogin`` set to ``yes``
 
-.. note:: For this example to work, you need the following on both servers:
-   - Ubuntu 16.04
-   - SSH
-   - PermitRootLogin set to "yes"
+Preparation
+~~~~~~~~~~~
 
-Preparation::
+Before you can perform a migration, you have to prepare.
+To do this, first make sure SSH is installed:
+
+.. code-block:: console
    
-   #     Install ssh
-   #
    apt install ssh -y
-   #
-   #     Edit ssh-config (enable root ssh login)
-   #
+
+Next, edit ssh-config and enable root ssh login
+
+.. code-block:: console
+   
    nano /etc/ssh/sshd_config
-   #
-   #     Change PermitRootLogin to "yes"
-   #
    PermitRootLogin yes
-   #
-   #     Restart ssh service
-   #
-   service ssh stop
-   #
-   service ssh start
-   #
-   #     Install ownCloud on new server
-
-=========
+
+Finally, restart SSH
+   
+.. code-block:: console
+   
+   service ssh restart
+
+Finally, you need to install ownCloud on the new server.
+
 Migration
-=========
-
-1. Put original server in maintenance mode::
-
-      #     Go in owncloud dir
-      #
-      cd /var/www/owncloud/
-      #
-      #     Switch to maintenance mode
-      #
-      sudo -u www-data php occ maintenance:mode --on
-      #
-      #     wait for 6-7 min and stop apache2
-      #
-      service apache2 stop
-
-2. Transfer the database::
-
-      #     Go in owncloud dir
-      #
-      cd /var/www/owncloud/
-      #
-      #     Backup the database
-      #
-      mysqldump --single-transaction -h localhost -u admin -ppassword owncloud > owncloud-dbbackup.bak
-      #
-      #     Export the database **to** new server
-      #
-      rsync -Aaxt owncloud-dbbackup.bak root@new_server_address:/var/www/owncloud 
-      #
-      #     Import the database **on** new server
-      #
-      mysql -h localhost -u admin -ppassword owncloud < owncloud-dbbackup.bak
-
-.. note:: You can find the values for the mysqldump command in your config.php at your owncloud directory.
-   [server]= dbhost, [username]= dbuser, [password]= dbpassword, and [db_name]= dbname.
-
-.. note:: For InnoDB tables only: The --single-transaction flag will start a transaction before running. 
+~~~~~~~~~
+
+Enable Maintenance Mode
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to enable maintenance mode. 
+to do that, use the following commands:
+
+.. code-block:: console
+
+    cd /var/www/owncloud/
+    sudo -u www-data php occ maintenance:mode --on
+
+After that's done, wait for 6-7 minutes and stop Apache:
+
+.. code-block:: console
+
+   service apache2 stop
+
+Transfer the Database
+^^^^^^^^^^^^^^^^^^^^^
+
+Now, you have to transfer the database from the old server to the new one.
+To do that, first backup the database.
+
+.. code-block:: console 
+
+    cd /var/www/owncloud/
+    mysqldump --single-transaction -h localhost -u admin -ppassword owncloud > owncloud-dbbackup.bak
```

Comment:
You can use it with or without a space but it could make sense to make it consitent between the -u and -h